### DISCUSSION TITLE
SSL Error locally (can't test it)

### DISCUSSION THREAD

**Discussion by philneumann** [spatie/laravel-discord-alerts#40] [Q&A]
I get the following SSL error if i want to try this out locally.
![294825863-54913e13-5178-4475-bad2-b81d37e4d02f](https://github.com/spatie/laravel-discord-alerts/assets/65663272/101503f1-aa38-4f10-9d35-656ec0fb20fb)

**Comment by philneumann** [comment]
@freekmurze 
Can you say something here as you moved it?

**Reply by freekmurze** [reply]
Nope, haven't come across this one myself, but I'm pretty sure it's not caused by our code.

If it is, feel free to submit a PR.
